neural_lam/models/ar_model.py

```diff
@@ -244,10 +286,15 @@ def validation_step(self, batch, batch_idx):
         val_log_dict = {
             f"val_loss_unroll{step}": time_step_loss[step - 1]
             for step in self.args.val_steps_to_log
+            if step < len(time_step_loss)
```

Comment:
Note that `args.val_steps_to_log` uses 1-indexing, so the indexing of `time_step_loss` is for `step - 1`. This should mean that we can allow
```suggestion
            if step <= len(time_step_loss)
```
e.g. if you roll out 1 step during training then `len(time_step_loss)=1` and we can include step 1 in `args.val_steps_to_log`. This should result in getting an entry `val_loss_unroll1` in `val_log_dict`.